### PR TITLE
Allow User to Set the File Format of Result Data

### PR DESCRIPTION
**Pull request by IsaacMilarky**
## Problem

Currently the script only outputs a static output file, "deduped_record_mapping.xlsx", in the directory that is passed into the CLI. 

## Solution

Change the usage of the CLI to make the user determine the file format of the output along with the filename. 

## Result

The usage of the CLI will change slightly from accepting a directory to place the output to accepting a file to place the output with dynamic determination of the format of the file. Most file formats supported by pandas are included. 

There will also need to be some integration with the front-end to make this change work.

### CODE REVIEW

File: frontend/main.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ function runProgram(filePath) {
     OPTIONS.FORMAT,
     identifyFormat(fileName),
     filePath,
-    currentDirectory + "/",
+    currentDirectory + "/" + RESULTS_SPREADSHEET,
```
Comment (decause-gov) [inline frontend/main.js:48]: rename this now that we support other formats?

**Review by decause-gov — APPROVED** [review]
LGTM +1 🚢

**Comment by natalialuzuriaga** [comment]
@IsaacMilarky As for frontend integration, I added a dropdown component where the user can select what file format they'd like to have the results data in with the 7 options provided in the backend. Tested, LGTM!

Screen Shot 2024-05-02 at 5 05 18 PM
Screen Shot 2024-05-02 at 5 05 30 PM

File: frontend/main.js (hunk 1 above)
Comment (natalialuzuriaga) [reply frontend/main.js]: Changed this variable to `RESULTS_FILE`

**Pull request merged**